### PR TITLE
Release minInstances feature

### PR DESCRIPTION
**Pull request by inlined**
🎉

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1 @@
+- Make the minInstances feature public
```
Comment (taeold) [inline CHANGELOG.md:1]:
```suggestion
- Make the minInstances feature public.
```

**Review by taeold — APPROVED** [review]

**Comment by jthegedus** [comment]
Does this require using v2 or is this also available in v1 now?

**Comment by taeold** [comment]
@jthegedus It's available on v1 namespace.

**Comment by jthegedus** [comment]
Great, thanks for the clarification

**Pull request merged**